tool/tctl/common/resource_command.go

```diff
@@ -604,6 +605,20 @@ func (rc *ResourceCommand) createInstaller(ctx context.Context, client auth.Clie
 	return trace.Wrap(err)
 }
 
+func (rc *ResourceCommand) createNode(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
+	server, err := services.UnmarshalServer(raw.Raw, "node")
```

Comment:
nit: is there a constant that can be used for `"node"`?

Reply:
There is! d589b9055ba7f0ef5e80cd55ade628d2442077a8